crates/holochain_metrics/src/lib.rs

```diff
@@ -147,13 +147,6 @@ pub enum HolochainMetricsConfig {
     },
 }
 
-const E_CHILD_SVC: &str = "HOLOCHAIN_INFLUXIVE_CHILD_SVC";
```

Comment:
If possible can unrelated changes be kept for another PR? Or is this clippy related?